### PR TITLE
Seft collection instrument

### PR DESCRIPTION
**Pull request by akssinhaamit**
# Motivation and Context

# What has changed

# How to test?

# Links

### CODE REVIEW

File: response_operations_ui/templates/collection_exercise/ce-instrument-section.html [hunk 1]
```diff
@@ -1,8 +1,71 @@
 <section class="ce-section {{'panel panel--simple panel--error' if missing_ci else ''}}">
+
+  {# 
+    This section needs to be updated once RAS-117 is completed.
+    Create new survey should provide an attribute, whether the survey is SEFT or EQ  
+  #}
+  {% set short_name = survey.shortName %}
```
Comment (insacuri) [inline response_operations_ui/templates/collection_exercise/ce-instrument-section.html:7]: Feel free to disagree with me, but I think this should be done in the view code rather in the template.  I think the aim is to keep logic in the templates to a minimum where possible and it'd be easier to do something like 
```
{% if seft %}
....
```

File: response_operations_ui/templates/survey.html [hunk 2]
```diff
@@ -83,16 +83,10 @@ <h1 name="page-survey-title" class="u-mb-l">{{survey.longName}}</h1>
...
-            {{
-                onsButton({
-                    "text": "Create collection exercise",
-                    "id": "create-collection-exercise",
-                    "name": "create-collection-exercise-link"
-                })
-            }}    
-    </form>
+      <p class="field u-mt-l">
+        <a href="{{ url_for('collection_exercise_bp.get_create_collection_exercise_form', survey_ref=survey.surveyRef, short_name=survey.shortName.replace(' ', '')) }}"
```
Comment (insacuri) [inline response_operations_ui/templates/survey.html:87]: Might need to check if links that look like buttons are allowed

**Review by insacuri — CHANGES_REQUESTED** [review]

File: tests/views/test_collection_exercise.py [hunk 3]
```diff
@@ -558,7 +567,7 @@ def test_choose_collection_instrument_when_first(self, mock_request, mock_detail
         response = self.client.get(f'/surveys/{short_name}/{period}', follow_redirects=True)
 
         self.assertEqual(response.status_code, 200)
-        self.assertIn("Add a collection instrument. Must be XLSX".encode(), response.data)
+        self.assertNotIn("Add a collection instrument. Must be XLSX".encode(), response.data)
```
[the remark this answers is not in the document]
Comment (akssinhaamit) [reply tests/views/test_collection_exercise.py:570]: Sure... makes sense
will change it

File: response_operations_ui/templates/collection_exercise/ce-instrument-section.html (hunk 1 above)
Comment (akssinhaamit) [reply response_operations_ui/templates/collection_exercise/ce-instrument-section.html:7]: I will agree with you, but this code has been done just till the time we get Bilals PR in place. We can still take this logic to the view. I will try to put this logic to view code.

**Review by insacuri — APPROVED** [review]

**Pull request merged**